examples/jaeger-hotrod/README.md

```diff
@@ -10,10 +10,10 @@ This demo will use the revered Jaeger Hotrod app. The purpose of this demo is to
 docker-compose up -d --build
 ``` 
 
-The above command will start the Jaeger Hotrod sample, Jaeger Agent, OpenTelemetry Collector, Data Prepper, OpenSearch and OpenSearch Dashboards. Wait for few minutes for all the containers to come up, the DataPrepper will restart until OpenSearch becomes available.
+The above command will start the Jaeger HotROD sample, Jaeger Agent, OpenTelemetry Collector, Data Prepper, OpenSearch and OpenSearch Dashboards. Wait for few minutes for all the containers to come up, the DataPrepper container will restart until OpenSearch becomes available.
 
-After successful start, 
+After the Docker image is running, do the following.
 
-* use the hot rod app at localhost:8080 
-* check the traces in OpenSearch Dashboards trace analytics dashboards at localhost:5601/app/trace-analytics-dashboards#/
+* Open the HotROD app at [http://localhost:8080](http://localhost:8080). Press the buttons in the UI to simulate requests. 
+* Load the OpenSearch Dashboards trace analytics dashboard at [http://localhost:5601/app/observability-dashboards#/trace_analytics/home](http://localhost:5601/app/observability-dashboards#/trace_analytics/home). You can view traces and the service map here.
```

Comment:
http://localhost:5601/app/observability-dashboards#/trace_analytics/home results in a 404.

http://localhost:5601/app/trace-analytics-dashboards#/ is the correct URL

Reply:
On my machine, the former is the correct URL and the latter gives a 404.

Reply:
This a change in OpenSearch 1.2.0. The former (and longer) URL is for OpenSearch 1.2.0 and above. The latter link is for OpenSearch 1.1.0 and below. I've updated the documentation to note that possibility.